packages/connect-disputable-voting/src/models/ERC20.ts

```diff
@@ -1,15 +1,34 @@
+import { miniMeAbi } from '@aragon/connect-core'
+import { BigNumber, Contract, providers as ethersProviders } from 'ethers'
+
 import { ERC20Data } from '../types'
 
 export default class ERC20 {
+  #provider: ethersProviders.Provider
```

Comment:
Should we name this `ethersProvider`? This is because we also have Ethereum providers (EIP-1193) in the library.